src/sonic-bgpcfgd/bgpcfgd/managers_static_rt.py

```diff
@@ -169,7 +169,9 @@ def __init__(self, af_id, blackhole, dst_ip, if_name, dist, vrf):
         self.ip = zero_ip(af_id) if dst_ip is None or dst_ip == '' else dst_ip
         self.interface = '' if if_name is None else if_name
         self.nh_vrf = '' if vrf is None else vrf
-        if self.blackhole != 'true' and self.is_zero_ip() and len(self.interface.strip()) == 0:
+        if not self.is_portchannel():
+            self.is_ip_valid()
```

Comment:
What is the purpose of these two lines? To make sure an exception is thrown if the IP is invalid?